### ISSUE TITLE
My 2 Nebra hotspots are not working

### ISSUE THREAD

**Issue by cixxxxx** [NebraLtd/helium-nebra-indoor1#4]
Can somebody help me please, i wrote to support, open a ticket but there is still no reply, i have 2 nebra miners which were working before, but now there is no activity from them, on one yellow and green light are on all the time, on second yelow is on and green is blinking. Not sure what i need to do please someone help me.
model nebhnt -hhrk4 -868
device s/n bfd0b29825cc83b7

**Comment by WayneNebra** [comment]
What's your ticket number as that miner needs flashing and I'll send the instructions for you.

**Comment by cixxxxx** [comment]
Hello finally somebody :)
 for 2 nebra miners ticket is [#443517]
And for 2 bobcat miners ticket is [#442935]
I can get them to work, please help :(

**Comment by WayneNebra** [comment]
Sent you the instructions for miner with s/n bfd0b29825cc83b7, please send the other one in on that ticket, Bobcats, you can only use the 1gig version with Nebra at the moment. (also sent you details on that ticket)

**Comment by cixxxxx** [comment]
i sent all bobcat details, nebra is the sn df9d79bb9a40b4cc

**Comment by cixxxxx** [comment]
@WayneNebra where are you, please check the tickets

**Comment by WayneNebra** [comment]
Flash **df9d79bb9a40b4cc** the same way :) 

I'll look at the Bobcat ticket tomorrow.

**Comment by cixxxxx** [comment]
already did the nebra, please look at the bobcat now, and tell me what to do, i dissasembly both of them, and waiting on you to tell me what to do, can i also flash it with sd card, and which file to use?

**Comment by cixxxxx** [comment]
@WayneNebra can you look at the bobcat ticket, tell me what do i need to do?
can i also flash it with sd card, and which file to use?
do i have to leave sd card in bobcat or do i do same like Nebra flash and than remove it.

**Comment by WayneNebra** [comment]
> @WayneNebra can you look at the bobcat ticket, tell me what do i need to do? can i also flash it with sd card, and which file to use? do i have to leave SD card in bobcat or do i do same like Nebra flash and than remove it.

I sent you a message on the Bobcat ticket on how you find out which version you have, and yes, if they are the 1 gig version then you would leave the SD card installed in the machine. 

Are you a member of the Bobcat Discord Channel? Full of great information. 

This is what they look like, the 1gig versions, [https://fccid.io/2AZCK-MINER300/Internal-Photos/Internal-photo-5373615​](https://fccid.io/2AZCK-MINER300/Internal-Photos/Internal-photo-5373615%E2%80%8B) 
​
1G 2AZCKMINER300  https://fccid.io/2AZCK

Bobcat Discord https://discord.gg/XX7beg64nR

**Comment by cixxxxx** [comment]
I can see any picture on this link that you sent??

**Comment by cixxxxx** [comment]
@WayneNebra  i managed to install nebra software on bobcat and they are working but only via lan cable, when i disconnect the lan and try to connect to wifi via bluetooth it founds miner but when i click pair it just gives me an error device is disconnected...
sending you picture here plese help??
![Screenshot_20231219_223543_Nebra](https://github.com/NebraLtd/helium-nebra-indoor1/assets/78624495/b9d7c90c-7da4-4afc-a59d-6a85b38ffc5d)

**Comment by WayneNebra** [comment]
Can you send me the JSON file on the ticket from local diagnostics if you like, also try connecting to the miner when you 1st power it on as BT runs for the 1st five minutes. (I'd just leave the LAN connected it won't do any harm) in fact after HIP83 you'd be better on LAN anyway. 

_HIP83 changed which miners are selected to receive rewards. Valid witnesses are no longer selected at random. With HIP83, the 14 "quickest" witnesses get the rewards. Miners on low-latency internet have an advantage._

**Comment by cixxxxx** [comment]
The problem is i cant use lan on this location.
I have to use wifi and when i try pair miner with bluetooth it gives me
error.
